api-report/purchases-js.api.md

```diff
@@ -376,6 +378,14 @@ export class Purchases {
     static setPlatformInfo(platformInfo: PlatformInfo): void;
 }
 
+// @public
+export interface PurchasesConfig {
+    apiKey: string;
+    appUserId: string;
+    flags?: FlagsConfig;

```

Comment:
I do wonder now whether we would like to flatten some of these properties into `PurchasesConfig`... But I think it's ok to keep it like this.

Reply:
Yeah... it depens on how many settings we would need... Let's keep it as is for now.